docs/source/mayavi/mlab_running_scripts.rst

```diff
@@ -37,31 +36,34 @@ Using together with Matplotlib's pylab
 If you want to use Matplotlib's pylab with Mayavi's mlab in IPython you
 should:
 
-    * if your IPython version is greater than 0.11: start IPython with::
+    * start IPython with::
 
-        $ ipython --gui=wx --pylab=wx
+        $ ipython --gui=qt --pylab=qt
 
-    * else, if your IPython version is greater than 0.8.4: start IPython with
-      the following options::
+    * elsewhere, start IPython with the `--gui=qtt` option::
 
-        $ ipython -pylab -wthread
-
-    * elsewhere, start IPython with the `-wthread` option::
-
-        $ ipython -wthread
+        $ ipython --gui=qt
 
       and **before** importing pylab, enter the following Python
       commands::
 
         >>> import matplotlib
-        >>> matplotlib.use('WxAgg')
+        >>> matplotlib.use('Qt4Agg')
         >>> matplotlib.interactive(True)
 
 If you want matplotlib and mlab to work together by default in IPython,
 you can change you default matplotlib backend, by editing the
 `~/.matplotlib/matplotlibrc` to add the following line::
 
-    backend     : WXAgg
+    backend     : Qt4Agg
+
+If for some reason, the Qt backend does not work, you can use the wx
+backend.  To do this you may do::
+
+  $ ETS_TOOLKIT=wx
+  $ ipython --gui=wx
+
+
```

Comment:
May I also suggest mentioning the fact that Mayavi cares about `ETS_TOOLKIT` and not the `--gui` switch of ipython?  Or is it mentioned somewhere else? For example, mayavi would use qt4 for the following setup (bizarre but possible, especially if the shell environment is set by someone else):
```
$ export ETS_TOOLKIT=qt
$ ipython --gui=wx
```